tools/release.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# fail if we encounter an error, uninitialized variable or a pipe breaks
+set -eu -o pipefail
+
+function die {
```

Comment:
Shouldn t we be using the "die" from "functions" file?

Reply:
fixed